README.md

```diff
@@ -8,13 +8,11 @@ To change your SSH server default port (i.e. 5001, see `rules.yaml`) and restart
 ```
 sed -i 's/Port 22/Port 5001/' /etc/ssh/sshd_config
 ```
-Download glutton, and install dependencies using `glide`:
+Download glutton, and install dependencies using `dep`:
 ```
 go get github.com/mushorg/glutton
 cd $GOPATH/src/github.com/mushorg/glutton/
-curl https://glide.sh/get | sh
-glide install
-glide update
+dep ensure
```

Comment:
Maybe add a line on how to install `dep`?

Reply:
I was going off what I found in the `Dockerfile` and `.travis.yml` files. It looks like it missed it in the `.travis.yml` file.

There are two way we can install it. The first is how you have it in `.travis.yml` `go get -u github.com/golang/dep/cmd/dep`. The other is `curl -fsSL -o /usr/local/bin/dep https://github.com/golang/dep/releases/download/vX.X.X/dep-linux-amd64 && chmod +x /usr/local/bin/dep`. It looks like the first way might be a bit more straightforward.

Reply:
yeah, first version is what they recommend. Maybe we want to fix the version we are pulling?

Reply:
We might want to. As per the FAQ for `dep`:
>Since dep is expected to change until v1.0.0 is released, it is recommended to rely on a released version. You can find the latest binary from the releases page.